src/electron/views/automated-checks/components/automated-checks-view.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+
+export type AutomatedChecksViewDeps = {};
+
+export type AutomatedChecksViewProps = {
```

Comment:
we're not currently using this, maybe we should drop it until we actually needed it (YAGNI)

Reply:
Removed in the most recent commit